app/models/user.rb

```diff
@@ -15,7 +15,12 @@ def approve!
   end
 
   def role?(role)
-    admin? || send(role)
+    admin? ||
+    (master_of_ceremonies? || scoring_enabled?) && send(role)
+  end
+
+  def scoring_enabled?
```

Comment:
This is the beginning of #61, too. If this method returns `false`, then only the "Master of Ceremonies" can modify scores. (It effectively takes all other non-admin roles away).